### ISSUE TITLE
fails to install lasted kernel with dkms compilation - write error: Broken pipe (scap)

### ISSUE THREAD

**Issue by mckaygerhard** [damentz/liquorix-package#157]
triying to install lasted kernel in bookworm fails with rare error during kdms compilation:
(scap log is attached), compilation of scap on debian normal kernel is fine!

```
Setting up linux-headers-6.7.2-3-liquorix-amd64 (6.7-4.1~bookworm) ...
/etc/kernel/header_postinst.d/dkms:
dkms: running auto installation service for kernel 6.7.2-3-liquorix-amd64.
/usr/sbin/dkms: line 2497: echo: write error: Broken pipe
Sign command: /lib/modules/6.7.2-3-liquorix-amd64/build/scripts/sign-file
Binary /lib/modules/6.7.2-3-liquorix-amd64/build/scripts/sign-file not found, modules won't be signed

Building module:
Cleaning build area...
make -j16 KERNELRELEASE=6.7.2-3-liquorix-amd64 -C /lib/modules/6.7.2-3-liquorix-amd64/build M=/var/lib/dkms/[make.log](https://github.com/damentz/liquorix-package/files/14125211/make.log)/0.1.1dev+git20220316.e5c53d64/build...(bad exit status: 2)
Error! Bad return status for module build on kernel: 6.7.2-3-liquorix-amd64 (x86_64)
Consult /var/lib/dkms/scap/0.1.1dev+git20220316.e5c53d64/build/make.log for more information.
Error! One or more modules failed to install during autoinstall.
Refer to previous errors for more information.
dkms: autoinstall for kernel: 6.7.2-3-liquorix-amd64 failed!
run-parts: /etc/kernel/header_postinst.d/dkms exited with return code 11
Failed to process /etc/kernel/header_postinst.d at /var/lib/dpkg/info/linux-headers-6.7.2-3-liquorix-amd64.postinst line 11.
dpkg: error processing package linux-headers-6.7.2-3-liquorix-amd64 (--configure):
 installed linux-headers-6.7.2-3-liquorix-amd64 package post-installation script subprocess returned error exit status 1
Setting up linux-image-6.7.2-3-liquorix-amd64 (6.7-4.1~bookworm) ...
I: /initrd.img is now a symlink to boot/initrd.img-6.7.2-3-liquorix-amd64
/etc/kernel/postinst.d/dkms:
dkms: running auto installation service for kernel 6.7.2-3-liquorix-amd64.
/usr/sbin/dkms: line 2497: echo: write error: Broken pipe
Sign command: /lib/modules/6.7.2-3-liquorix-amd64/build/scripts/sign-file
Binary /lib/modules/6.7.2-3-liquorix-amd64/build/scripts/sign-file not found, modules won't be signed

Building module:
Cleaning build area...
make -j16 KERNELRELEASE=6.7.2-3-liquorix-amd64 -C /lib/modules/6.7.2-3-liquorix-amd64/build M=/var/lib/dkms/scap/0.1.1dev+git20220316.e5c53d64/build...(bad exit status: 2)
Error! Bad return status for module build on kernel: 6.7.2-3-liquorix-amd64 (x86_64)
Consult /var/lib/dkms/scap/0.1.1dev+git20220316.e5c53d64/build/make.log for more information.
Error! One or more modules failed to install during autoinstall.
Refer to previous errors for more information.
dkms: autoinstall for kernel: 6.7.2-3-liquorix-amd64 failed!
run-parts: /etc/kernel/postinst.d/dkms exited with return code 11
dpkg: error processing package linux-image-6.7.2-3-liquorix-amd64 (--configure):
 installed linux-image-6.7.2-3-liquorix-amd64 package post-installation script subprocess returned error exit status 1
dpkg: dependency problems prevent configuration of linux-image-liquorix-amd64:
 linux-image-liquorix-amd64 depends on linux-image-6.7.2-3-liquorix-amd64 (= 6.7-4.1~bookworm); however:
  Package linux-image-6.7.2-3-liquorix-amd64 is not configured yet.

dpkg: error processing package linux-image-liquorix-amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of linux-headers-liquorix-amd64:
 linux-headers-liquorix-amd64 depends on linux-headers-6.7.2-3-liquorix-amd64 (= 6.7-4.1~bookworm); however:
  Package linux-headers-6.7.2-3-liquorix-amd64 is not configured yet.

dpkg: error processing package linux-headers-liquorix-amd64 (--configure):
 dependency problems - leaving unconfigured
```

**Comment by mckaygerhard** [comment]
workaround we cannot use the scap module for sysdig system calls capturing, deinstall falcosecurity-scap-dkms and use direct git clone for or dont use it. with liquoris

**Comment by damentz** [comment]
Can you try the latest kernel once available?  6.7-6 / 6.7.3-2

**Comment by damentz** [comment]
The issue here appears to have been a combination of two issues:
1. The GCC latent entropy plugin started failing for mismatched GCC versions even when that's not the case.  Since we don't _need_ it, dropping that was easy.
2. Second, a stable update changed code that the nvidia driver needed, causing its packages to fail to build due to GPL symbols for RCU read (un)lock.  Making those non-GPL for now will help until kernel 6.8.

Marking this as resolved.
